### ISSUE TITLE
Can't install the cert-manager-trust Helm chart

### ISSUE THREAD

**Issue by nuriel77** [cert-manager/trust#5]
Followed the instruction on installing the Helm chart. No success:
```
$ helm repo add jetstack https://charts.jetstack.io --force-update
"jetstack" has been added to your repositories

$ helm upgrade -i -n cert-manager cert-manager-trust jetstack/cert-manager-trust --wait
Release "cert-manager-trust" does not exist. Installing it now.
Error: failed to download "jetstack/cert-manager-trust" (hint: running `helm repo update` may help)

$ helm repo update
Hang tight while we grab the latest from your chart repositories...
...
...Successfully got an update from the "jetstack" chart repository
...
Update Complete. Happy Helming!

$ helm search repo jetstack
NAME                                    CHART VERSION   APP VERSION     DESCRIPTION
jetstack/cert-manager                   v1.6.1          v1.6.1          A Helm chart for cert-manager
jetstack/cert-manager-approver-policy   v0.1.0          v0.1.0          A Helm chart for cert-manager-approver-policy
jetstack/cert-manager-csi-driver        v0.2.0          v0.2.0          A Helm chart for cert-manager-csi-driver
jetstack/cert-manager-csi-driver-spiffe v0.1.0          v0.1.0          A Helm chart for cert-manager-csi-driver-spiffe
jetstack/cert-manager-istio-csr         v0.3.1          v0.3.0          A Helm chart for istio-csr
```

**Comment by nuriel77** [comment]
I proceeded by cloning the repository and installing the chart from a local directory.
However, I cannot pull the image.
```
$ docker pull quay.io/jetstack/cert-manager-trust
Using default tag: latest
Error response from daemon: unauthorized: access to the requested resource is not authorized
```

**Comment by JoshVanL** [comment]
Thanks for opening @nuriel77, apologies for not having an image available. After #6 is merged, I'll release the v0.1.0 image and publish the helm chart.

**Comment by JoshVanL** [comment]
@nuriel77 the `v0.1.0` image and helm chart has now been released so you should be able to install trust using Jetstack's Helm repository.

Feel free to re-open the issue if you are still having issues.

/close
